oauthproxy.go

```diff
@@ -617,6 +640,25 @@ func getRemoteAddr(req *http.Request) (s string) {
 	return
 }
 
+func getRealIP(req *http.Request) (*net.IP, error) {
+	realIP := req.Header.Get("X-Real-IP")
```

Comment:
X-Forwarded-For ?

Reply:
Cloudflare passes us `X-Real-IP` don't they?  Otherwise we'd be getting the `X-Forwarded-For` IP of cloudflare

Reply:
Add a parameter?

Reply:
Yeah that's what I meant when I said `I think they’ll want a more flexible way to select how what is considered the real source IP, but we use X-Real-IP headers, right?`